### PR TITLE
Implement #657

### PR DESCRIPTION
**Pull request by vEnhance**
Everything here is up for editing.

- I put a fencepost at bluffs (~100 games) and priority (~200 games).
- I put a "rule of thumb" of 10-15 games per level before bluffs and 15-25 games per level afterwards.

### CODE REVIEW

File: docs/level-18.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ import LoadClue from '@site/image-generator/yml/level-18/load-clue.yml';
 import PriorityFinesseSpecial from '@site/image-generator/yml/level-18/priority-finesse-special.yml';
 import TrustFinesse from '@site/image-generator/yml/level-18/trust-finesse.yml';
 
-- Level 18 strategies should only be learned if you have 320+ games of experience with the group.
+- This level introduces *Priority*, which requires players to pay attention to the order that cards are played in.
+- This can take a while to internalize, because cues from *Priority* are easy to "miss".
```
Comment (DarthGandalf) [inline docs/level-18.md:14]: cues or clues?

Comment (vEnhance) [reply docs/level-18.md:14]: I did mean "cues". Would a different word be better? ("signals"?)

Comment (Zamiell) [reply docs/level-18.md:14]: signals is better

File: docs/level-10.md [hunk 2]
```diff
@@ -16,7 +16,10 @@ import ColorConnectRainbow from '@site/image-generator/yml/level-10/color-connec
 import ColorConnectRainbowFinesseImpossible from '@site/image-generator/yml/level-10/color-connect-rainbow-finesse-impossible.yml';
 import BluffPromptsPromptBluffs from '@site/image-generator/yml/level-10/bluff-prompts-prompt-bluffs.yml';
 
-- Level 10 strategies should only be learned if you have 160+ games of experience with the group.
+- This level introduces *Bluffs*, a way to tell a fib to the next player to cause them to blind-play.
```
Comment (Zamiell) [inline docs/level-10.md:19]: fib should be `*Fib*`

File: docs/level-18.md (hunk 1 above)
Comment (vEnhance) [reply docs/level-18.md]: Sounds good

**Comment by vEnhance** [comment]
Edits look good to me

**Comment by Zamiell** [comment]
thanks

**Pull request merged**